doc/source/examples/incompressible-flow/3d_mixer_using_single_rotating_frame/3d_mixer_using_single_rotating_frame.rst

```diff
@@ -0,0 +1,496 @@
+
+.. image:: images/velocity_magnitude.png
+   :alt: Ribbon Mixer Velocity magnitude
+   :align: center
+   :name: Ribbon Mixer Velocity magnitude
+
+In the reference frame of the tank, the velocity field and its streamlines are visualized in an Eulerian frame. It is possible to do so by simply selecting ``velocity_eulerian`` in Paraview.
+
+.. image:: images/velocity_eulerian.png
+   :alt: Ribbon Mixer Velocity in an Eulerian frame
+   :align: center
+   :name: Ribbon Mixer Velocity in an Eulerian frame
+
+Generating :math:`N_p` vs :math:`Re` Curves :math:`(Re \in [0.1, 100])`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Copying Files to Your Local Computer (Using SFTP)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before postprocessing the results of the simulations, you may want to copy the files from the remote machine to your local one. You may do so by connecting to the remote computer with the Secure File Transfer Protocol (SFTP):
+
+.. code-block:: text
+
+    sftp username@clustername.computecanada.ca
+
+With the ``get`` command you may copy the remote ``Np_vs_Re`` folder:
+
+.. code-block:: text
+
+    get -r Path/To/Remote_Np_vs_Re Path/To/Local_directory
+
+.. note::
+
+    Earlier, before launching the simulations, we could have also uploaded the initial ``Np_vs_Re`` folder using this method with the ``put`` command:
+
+    .. code-block:: text
+
+        put -r Path/To/Local_Np_vs_Re Path/To/Remote_scratch_directory
+
+Once the transfer is completed, you may exit with the ``exit`` command.
+
+Postprocess Data
+^^^^^^^^^^^^^^^^^
+In order to generate the :math:`N_p` vs :math:`Re` curves, we must first gather all the calculated :math:`\Gamma` from all different cases. You may do so by executing the ``gather_torques.sh`` Bash script. A new ``gather.dat`` file will be generated containing the number of cells and the :math:`\Gamma` components of each simulated case.
+
+.. note::
+
+    In order to run the ``gather_torques.sh`` script, you must make sure that the script is executable. If it is not executable, you may add the permissions with:
+
+    .. code-block:: text
+
+        chmod +x Path/To/gather_torques.sh
+
+After that, you may run the ``plot_Np_vs_Re.py`` Python script to get the figure shown below.
+
+.. image:: images/Np-Re.png
+   :alt: Power curve
+   :align: center
+   :name: Power curve
+
+As you can see, :math:`N_p` decreases when :math:`Re` increases. It is explained by:
+
+.. math::
+
+    N_p = \frac{2 \pi \Gamma}{\rho N^2 D^5} =\frac{1}{Re}  \frac{2 \pi \Gamma}{\mu  N D^3}
+
+As we can see in the image above, for :math:`Re < 10` (laminar regime), the Lethe curve fits well with the experimental values.
+
+References
```

Comment:
I would add a possibility for extension to suggest simulating it at a higher Reynolds number in a transient simulation and compare with the experimental data :).